README.md

```diff
@@ -1 +1,103 @@
-# openapi-msw
+<h1 align="center">OpenApi-MSW</h1>
+
+A tiny, type-safe wrapper around [MSW](https://mswjs.io) to add support for full
+type inference from OpenApi schema definitions that are generated with
+[OpenApi-TS](https://openapi-ts.pages.dev/introduction).
+
+## Installation
+
+You can install OpenApi-MSW with this shell command:
+
+```bash
+npm i -D openapi-msw
+```
+
+**Note:** This package has a peer-dependency to MSW **v2**. There is no plan to
```

Comment:
Opinionated structuring idea: Add a "Prerequisites" section before installation, containing this note and the first paragraph of the usage guide. That way, its clear what needs to be done before installing this package makes sense. Also, the "Usage Guide > Getting Started" subsection title might obsolete then.

Reply:
I am considering other changes to the docs as well, such as a list of key features/achievements in in the beginning and and outlining the motivation behind openapi-msw, i.e. errors for spec conflicts across the stack (app + test code). I will keep this idea in mind and come back to it when I make other improvements to the docs as well.